ingestion/src/metadata/ingestion/source/database/azuresql/metadata.py

```diff
@@ -23,6 +27,134 @@
 from metadata.ingestion.source.database.common_db_source import CommonDbSourceService
 
 
+class NCHAR(String):
+    """The SQL NCHAR type."""
+
+    __visit_name__ = "NCHAR"
+
+
+class NVARCHAR(String):
+    """The SQL NVARCHAR type."""
+
+    __visit_name__ = "NVARCHAR"
+
+
+class NTEXT(String):
+    """The SQL NTEXT type."""
+
+    __visit_name__ = "NTEXT"
+
+
+class BINARY(String):
+    """The SQL BINARY type."""
+
+    __visit_name__ = "BINARY"
+
+
+class IMAGE(String):
+    """The SQL IMAGE type."""
+
+    __visit_name__ = "IMAGE"
+
+
+class BIT(String):
+    """The SQL BIT type."""
+
```

Comment:
Let's have all the classes for data types under datatypes.py
for azuresql, 
ingestion/src/metadata/ingestion/source/database/azuresql/datatypes.py